gradle-baseline-java/src/test/resources/com/palantir/baseline/build.gradle

```diff
@@ -16,8 +16,8 @@ dependencies {
 
 tasks.withType(Checkstyle) {
```

Comment:
this actually isn't required until the gradle8 move but since I'd already done it just left it in.